manifests/gateways/istio-ingress/values.yaml

```diff
@@ -92,6 +92,9 @@ gateways:
     - port: 15011
```

Comment:
Please also remove 15011 and 8060 - no longer needed. ( there is a quota on firewall rules and/or ports on some clouds)

Reply:
fixed. I also remove the unused virutalservice and destinationrules for pilot and citadel